l-sgov/l-sgov-359-2011-diagram.ttl

```diff
@@ -0,0 +1,34 @@
+@prefix rdf:   <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix owl:   <http://www.w3.org/2002/07/owl#> .
+@prefix l-sgov-359-2011-pojem: <http://onto.fel.cvut.cz/ontologies/slovník/legislativní/sbírka/359/2011/pojem/> .
```

Comment:
Nemělo by to používat prefixy z domény slovník.gov.cz?

Reply:
Ano, musí to být https://slovník.gov.cz/ místo http://onto.fel.cvut.cz/ontologies/slovník/ . Čili je nutno přenastavit v modeliu u package reprezentující slovník vlastnost Vocabulary IRI Base (IRI).

Reply:
Čistě technicky, nemělo by to tady bejt úplně jedno?